gamedata/movedefs.lua

```diff
@@ -244,8 +244,8 @@ local moveDatas = {
 		badslope = 22,
 		badwaterslope = 255,
 		crushstrength = 252,
-		footprintx = 4,
-		footprintz = 4,
+		footprintx = 5,
+		footprintz = 5,
```

Comment:
Shouldn't this be called HOVER5 now? It seemed the name was intended to match the foorptinr size,